server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -558,18 +558,18 @@ public void testInsertFromQueryGlobalAggregate() throws Exception {
         execute("refresh table locations");
 
         execute("create table aggs (" +
-                " c long," +
+                " c double," +
                 " s double" +
                 ") with (number_of_replicas=0)");
         ensureYellow();
 
-        execute("insert into aggs (c, s) (select count(*), sum(position) from locations)");
+        execute("insert into aggs (c, s) (select avg(position), sum(position) from locations)");
```

Comment:
1. count is also an aggregation but was not failing. Do I get it right, that it's kinda exceptional and could work with source lookup?
2. since we updated this test, I wonder if we need a new one then. no strong opinion though.

Reply:
Good point, i think one is enough.